src/components/AccessKeyForm.js

```diff
@@ -158,7 +159,7 @@ const AccessKeyForm = ({ initialAccountId = null, minimal = false }) => {
           ) {
             setErrorMessage(err.data);
           } else {
-            setErrorMessage(__("An error happended."));
+            setErrorMessage(__("Unable to use access key."));
```

Comment:
@zackkatz This is the fall back message if response doesn't have a useable error message. I'm about to fix WHY this error was used, but if you want to have a different fallback message, let me know.

Reply:
"There was an error processing the access key."

Reply:
👍